src/modules/shared/components/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { Tabs } from '@meemoo/react-components';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+
+import { Icon } from '../Icon';
+
+const mockTabs = [
```

Comment:
Mag in een `__mocks__` folder

Reply:
Ik zet dat meestal pas in die folder als ik het op meerdere plekken ga nodig hebben (stories, test, ...) maar mss wel goed dat we hier een lijn in trekken en dat gwn standaard doen. Wordt al gefixed in eeen volgende PR 👍